### DISCUSSION TITLE
mobile app (android/ios) for setting ?

### DISCUSSION THREAD

**Discussion by singlag** [ExpressLRS/ExpressLRS#1344] [Ideas]
is it possible to make a mobile app (android/ios) connect to expresslrs TX via bluetooth for setting (expresslrs and opentx setting) ?

*Instead of commenting "I want this too", consider leaving a :+1: instead, to reduce spam and clutter*

**Comment by pkendall64** [comment]
This could more easily be done by providing a web-based UI off the existing update webserver in the TX module.

**Comment by MartinHugh** [comment]
Much simpler to write and maintain a web page I would expect.

In the same way that you can flash the TX in the field from your smartphone over WiFi
